### DISCUSSION TITLE
Umi3和Umi4是否可以使用module federation实现互相导出模块

### DISCUSSION THREAD

**Discussion by foxmaizi** [umijs/umi#11973] [Q&A]
目前遇到的问题是Umi4/max使用自带的mf功能，Umi3使用chainWebpack配置可以实现Umi4项目使用Umi3导出的模块，但是Umi3没法使用Umi4/max导出的模块

**Comment by fz6m** [comment]
1. umi 3 不维护了，建议升级到 umi 4 。

2. 由于 mf 作者表示会对 mf 近期进行比较大的调整和升级，所以以往的 mf 在未来可能造成严重的历史债，不建议继续使用现有的 mf 方案。
